Resources/Prototypes/Entities/Objects/Decoration/flora.yml

```diff
@@ -0,0 +1,292 @@
+- type: entity
+  id: BaseTree
+  abstract: true
+  components:
+  - type: Clickable
+  - type: Sprite
+    sprite: Objects\Decoration\Flora\flora_trees.rsi
```

Comment:
Should have a placementmode to default to tile centre even if it's not anchored.